Makefile

```diff
@@ -250,16 +249,20 @@ else
 	scp $(ASIC_USER)@$(ASIC_SERVER):$(REMOTE_ROOT_DIR)/$(ASIC_DIR)/synth/*.txt $(ASIC_DIR)/synth
 endif
 
-asic-sim-synth:
-	make -C $(FIRM_DIR) run BAUD=$(HW_BAUD)
+asic-sim-synth: sim-clean
+	make -C $(FIRM_DIR) run BAUD=$(SIM_BAUD)
+	make -C $(BOOT_DIR) run BAUD=$(SIM_BAUD)
 ifeq ($(shell hostname), $(ASIC_SERVER))
-	make -C $(ASIC_DIR) sim INIT_MEM=$(INIT_MEM) USE_DDR=$(USE_DDR) RUN_DDR=$(RUN_DDR) TEST_LOG=$(TEST_LOG) VCD=$(VCD) BAUD=$(HW_BAUD) SYNTH=1
+	make -C $(HW_DIR)/simulation/ncsim run INIT_MEM=$(INIT_MEM) USE_DDR=$(USE_DDR) RUN_DDR=$(RUN_DDR) TEST_LOG=$(TEST_LOG) VCD=$(VCD) BAUD=$(SIM_BAUD) SYNTH=1
```

Comment:
thiis is called SIM_DIR

Reply:
Not quite. If you run with SIMULATOR=ncsim, yes, it is SIM_DIR. If you run without SIMULATOR defined, it is the default (icarus). So I did as you see in the pull request.